src/runtime/pkg/oci/utils.go

```diff
@@ -1254,8 +1266,8 @@ func CalculateContainerSizing(spec *specs.Spec) (numCPU, memSizeMB uint32) {
 	return calculateVMResources(period, quota, memory)
 }
 
-func calculateVMResources(period uint64, quota int64, memory int64) (numCPU, memSizeMB uint32) {
-	numCPU = vcutils.CalculateVCpusFromMilliCpus(vcutils.CalculateMilliCPUs(quota, period))
+func calculateVMResources(period uint64, quota int64, memory int64) (numCPU float32, memSizeMB uint32) {
+	numCPU = vcutils.CalculateMilliCPUs(quota, period)
```

Comment:
What happened to `CalculateVCpusFromMilliCpus()` ?

Reply:
From what I gather, the return value is now a `float32` and it's propagated all the way up to `config.SandboxCPUs`. So I think that removing it here is legit.

Reply:
The actual rounding happens by calling `RoundUpNumVCPUs` from `NumVCPUs()`, which is now a function.

Reply:
The function `CalculateVCpusFromMilliCpus` is not needed anymore. I think it should be removed (along with the associated tests, which has a typo in the name anyway).